src/async-wrap.cc

```diff
@@ -181,6 +183,33 @@ static void Initialize(Local<Object> target,
 }
 
 
+void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
+  Environment* env = Environment::from_destroy_ids_idle_handle(handle);
```

Comment:
Forget everything I said about `SealHandleScope` not catching handles from an async call. My test case was incorrect. It does catch all leaked handles.